internal/model/sharedpullerstate.go

```diff
@@ -250,7 +250,7 @@ func (s *sharedPullerState) Progress() *pullerProgress {
 		CopiedFromElsewhere: s.copyTotal - s.copyNeeded - s.copyOrigin,
 		Pulled:              s.pullTotal - s.pullNeeded,
 		Pulling:             s.pullNeeded,
-		BytesTotal:          files.BlocksToSize(total),
-		BytesDone:           files.BlocksToSize(done),
+		BytesTotal:          db.BlocksToSize(total),
```

Comment:
This perhaps should become part of protocol?
Though I feel you recently moved it from protocol to files?